### PR TITLE
Updates for Sprint week 2024

### PR DESCRIPTION
**Pull request by arunkannawadi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,240 +2,251 @@
...
+                                   title="link to LSSTC Slack">#desc-hack-sprint</a>
+                              for Sprint-related announcements.
+                         </li>
+                         <li>
+                              <a href="https://lsstc.slack.com/archives/C01EY4AK7UJ"
+                                   title="link to LSSTC Slack">#desc-sprint-hotline</a>
+                              to report any technical problems that may arise during Sprint Day.
+                         </li>
+                         <!-- <li>
+                              <a href="https://lsstc.slack.com/archives/C05NLP9DTK8"
```
Comment (sethdigel) [inline index.html]: This should be removed or changed to a new channel.  I suspect that #desc-hack-sprint above may be enough

File: index.html [hunk 2]
```diff
@@ -246,154 +257,175 @@ <h2>Computing resources, datasets etc.</h2>
...
+          <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+          <!-- Social media -->
+          <h2>Social media</h2>
+          <!-- -->
+          <div class="item">
+               <div class="box">
+                    <input type="checkbox">
+               </div>
+               <div class="text">If you use social media professionally, please use the
+                    <a href="https://twitter.com/hashtag/lsstdesc" title="link to Twitter">#lsstdesc</a>
```
Comment (sethdigel) [inline index.html]: I do not know how Twitter works/worked but I wonder if this URL should be X

File: index.html [hunk 3]
```diff
@@ -246,154 +257,175 @@ <h2>Computing resources, datasets etc.</h2>
...
-     <script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
-     <script>
-     const cookie_key = "lsst_desc_pre_meeting_checklist_Feb2024";
-     const get_progress = function () {
-          return (0.5 + 99.5 / $("input").length * $("input:checked").length);
-     }
+          <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+          <script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
+          <script>
+               const cookie_key = "lsst_desc_pre_meeting_checklist_Oct2022";
```
Comment (sethdigel) [inline index.html]: Maybe it doesn't matter but suggest Oct2024 instead of Oct2022

**Review by sethdigel — CHANGES_REQUESTED** [review]
Here are a few minor comments for your consideration

File: index.html (hunk 1 above)
Comment (arunkannawadi) [reply index.html]: I suspect that channel is too crowded for all the chatter related to local activities. I've created a new channel and linked here.

File: index.html (hunk 2 above)
Comment (arunkannawadi) [reply index.html]: It gets automatically redirected, but I've updated the links.

**Review by sethdigel — APPROVED** [review]
Thank you, Arun!

**Pull request merged**